rest/model/distro.go

```diff
@@ -362,7 +361,6 @@ type APIDistro struct {
 	Provider              *string                  `json:"provider"`
 	ProviderSettings      map[string]interface{}   `json:"settings"`
 	ProviderSettingsList  []*birch.Document        `json:"provider_settings"`
-	ImageID               *string                  `json:"image_id"`
```

Comment:
This looks like a holdover from before we exposed all of ProviderSettings in the API.